Makefile

```diff
@@ -435,9 +432,6 @@ KUBERNETES_VERSION ?= v1.21.1
 BATS_VERSION ?= 1.2.1
 OLM_VERSION ?= v0.18.2
 
-# Use the vendored directory
-GOFLAGS = -mod=vendor
```

Comment:
why?

Reply:
It's because we aren't using the vendor directory any more so we want to remove this flag being passed. We want to operate in "normal" Go module mode.